public/locales/en-US/translations.json

```
@@ -115,18 +115,15 @@
   "close_request": "close channel request",
   "renew_channel": "renew chanel",
   "payment_channel_closed": "payment channel closed",
-  "paychannel_node_line1":
-    "It <1><0>{{action}}</0></1> a PayChannel node from <3><0>{{account}}</0></3> to <5><0>{{counterAccount}}</0></5>",
-  "paychannel_amount_changed":
-    "Amount changed by <1><0>{{difference}}</0><1><0>{{currency}}</0></1></1> from <3><0>{{previous}}</0><1><0>{{currency}}</0></1></3> to <5><0>{{final}}</0><1><0>{{currency}}</0></1></5>",
-  "paychannel_balance_changed":
-    "Balance changed by <1><0>{{difference}}</0><1><0>{{currency}}</0></1></1> from <3><0>{{previous}}</0><1><0>{{currency}}</0></1></3> to <5><0>{{final}}</0><1><0>{{currency}}</0></1></5>",
+  "paychannel_node_line1": "It <1><0>{{action}}</0></1> a PayChannel node from <3><0>{{account}}</0></3> to <5><0>{{counterAccount}}</0></5>",
```

nit: to follow the naming_convention used
```suggestion
  "paychannel_node_line_1": "It <1><0>{{action}}</0></1> a PayChannel node from <3><0>{{account}}</0></3> to <5><0>{{counterAccount}}</0></5>",
```

---

(Or you can change the naming convention below - either way consistency is nice :) )

---

(Actually now that I see there's a bunch of them it's probably not worth doing in this PR - maybe worth another one though)